anomalib/models/components/dimensionality_reduction/pca.py

```diff
@@ -19,7 +19,7 @@
 import torch
 from torch import Tensor
 
-from anomalib.core.model.dynamic_module import DynamicBufferModule
```

Comment:
Similar to `AnomalibModule`, location of `DynamicBufferModule` and access to it could be reconsidered